### DISCUSSION TITLE
Namespacing of modules with popular names

### DISCUSSION THREAD

**Discussion by prange** [dillonkearns/elm-pages#362] [General]
(Original start of discussion here: https://elmlang.slack.com/archives/CNSNETV37/p1674742549431329)

I recently came across a namespace collision for the `Path` module. It turns out the term _Path_ is popular.

One solution could be give the module a less popular name, like _UrlPath_, _PathUtils_ etc.
Another could be to give `Path` a prefix like Pages.Path

**Comment by prange** [comment]
@dillonkearns Do you think this is within the scope of v3?

**Reply by dillonkearns** [reply]
Yeah, I think it would be good to decide on a new name for the v3 release to avoid breaking changes after the stable release 👍

**Reply by dillonkearns** [reply]
Alright, this is now done! I went with `UrlPath` 👍
